server/conf/cms/secrets.sample.py

```diff
@@ -1,31 +1,250 @@
+- The following VALUES for new elements set in the configuration object must exist in the portal css as well:
+    - Any new selectors or css styles (add to /taccsite_cms/static/site_cms/css/src/_imports/branding_logos.css)
+    - Image files being references (add to /taccsite_cms/static/site_cms/img/org_logos)
+
+Values to populate:
+
+_SETTING_NAME = [                 # The name of the branding or logo config setting object.
+    "org_name",                         # The name of the organization the branding belongs too.
+    "img_file_src",                      # Path and filename relative to the static files folder.
+    "img_element_classes",        # The list of selectors to apply to the rendered element, these need to exist in the css/scss.
+    "a_target_url",                      # The href link to follow when clicked, use "/" for portal logos.
+    "a_target_type",                    # The target to open the new link in, use _blank for external links, _self for internal links.
+    "alt_text",                             # The text to read or render for web assistance standards.
+    "cors_setting",                      # The CORS setting for the image, set to anonymous by default.
+    "visibility"                             # Toggles wether or not to display the element in the template, use True to render, False to hide.
+]
+
+Branding Configuration Example.
+
+_ANORG_BRANDING = [
+   "anorg",
+   "site_cms/img/org_logos/anorg-logo.png"
+   "branding-anorg",
+   "https://www.anorg.com/"
+   "_blank",
+   "ANORG Logo",
+   "anonymous",
+   "True"
+]
+
+Logo Configuration Example.
+
+_ANORG_LOGO = [
+   "anorg",
+   "site_cms/img/org_logos/anorg-logo.png"
+   "branding-anorg",
+   "/"
+   "_self",
+   "ANORG Logo",
+   "anonymous",
+   "True"
+]
+"""
+
+########################
+# BRANDING
+
+_TACC_BRANDING = [
+    "tacc",
+    "site_cms/img/org_logos/tacc-white.png",
+    "branding-tacc",
+    "https://www.tacc.utexas.edu/",
+    "_blank",
+    "TACC Logo",
+    "anonymous",
+    "True"
+]
+
+_UTEXAS_BRANDING =  [
+    "utexas",
+    "site_cms/img/org_logos/utaustin-white.png",
+    "branding-utaustin",
+    "https://www.utexas.edu/",
+    "_blank",
+    "University of Texas at Austin Logo",
+    "anonymous",
+    "True"
+]
+
+_NSF_BRANDING = [
+    "nsf",
+    "site_cms/img/org_logos/nsf-white.png",
+    "branding-nsf",
+    "https://www.nsf.gov/",
+    "_blank",
+    "NSF Logo",
+    "anonymous",
+    "True"
+]
+
+# _BRANDING = [ _TACC_BRANDING, _UTEXAS_BRANDING ]        # Default TACC Portal.
+_BRANDING = [ _NSF_BRANDING, _TACC_BRANDING, _UTEXAS_BRANDING ]       # NSF Funded TACC Portal.
+
+########################
+# LOGOS
 
-_GOOGLE_ANALYTICS_PROPERTY_ID = ''
-_GOOGLE_ANALYTICS_PRELOAD = ''
+_PORTAL_LOGO =  [
+    "frontera",
+    "frontera-cms/img/org_logos/frontera-white-solo.png",
+    "",
+    "/",
+    "_self",
+    "Frontera Logo",
+    "anonymous",
+    "True"
+]
```

Comment:
@tacc-wbomar @taoteg Out of scope for this PR, but any reason this is not an object with descriptive keys? Makes it difficult to parse, and order appears to matter.

Reply:
There is no reason. John did not think of it when he authored that code originally, and we both agreed, after I implemented `LINKS (for Portal).` that object/keys would be better (and remove a `templatetag`).

Reply:
Yep, we can def upgrade this implementation.

Reply:
[GitLab Issue #53](https://gitlab.tacc.utexas.edu/wma-cms/cms-site-template/issues/53).

_**Notice**: I will migrate GitLab issues out of GitHub. I want first to research issue templates (which [I also want to do for JIRA](https://tacc-team.slack.com/archives/GQW4Q8HLG/p1600197732087300?thread_ts=1600103670.046500&cid=GQW4Q8HLG))._